### PR TITLE
Hide template stats in browse views

### PR DESCRIPTION
**Pull request by phlax**
fixes https://github.com/translate/pootle/issues/4568

### CODE REVIEW

**Comment by dwaynebailey** [comment]
lgtm just from visual/clicky testing.

Some observations, take or leave or flag as future issue:

* Do we want a progressbar on Project view? Indicate completeness and we don't have any concept of completeness for Templates
* We don't have expanded stats, OK.  But that means we can't see words vs units.  I realise we don't have that in normal.  But would be nice to have words and unit counts for templates.
* Templates TP listed when not logged in (shouldn't we hide it?)

**Comment by phlax** [comment]
hi @dwaynebailey the first two points seem like new features and outside the scope of this PR

re
> Templates TP listed when not logged in (shouldn't we hide it?)

i wondered about this - but then i dont really see why - i think it could be useful and its not private data - not sure tho.

**Comment by phlax** [comment]
maybe i misunderstood the first point - grokking

File: pootle/apps/pootle_data/utils.py [hunk 1]
```diff
@@ -423,7 +423,11 @@ def aggregate_children(self, stats):
         latest = dict(last_submission=None, last_created_unit=None)
         last_submission_pk = None
         last_created_unit_time = None
-        for child in stats["children"].values():
+        for path, child in stats["children"].items():
+            if path.startswith("templates-") or path == "templates":
+                if not self.context.pootle_path.startswith("/templates/"):
```
Comment (ta2-1) [inline pootle/apps/pootle_data/utils.py]: @phlax can we use something like `self.context.is_template()` here?

File: pootle/apps/pootle_project/views.py [hunk 2]
```diff
@@ -110,6 +110,12 @@ class ProjectBrowseView(ProjectMixin, PootleBrowseView):
     view_name = "project"
 
     @property
+    def is_templates_context(self):
+        return (
+            len(self.object_children) == 1
+            and self.object_children[0]["code"] == "templates")
```
Comment (ta2-1) [inline pootle/apps/pootle_project/views.py]: @phlax I'm not sure I'm clear with logic here. Can you describe it?

File: pootle/apps/pootle_data/utils.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_data/utils.py]: not really because its not always a template - it can be a directory or a tp etc

**Comment by unho** [comment]
lgtm

File: pootle/apps/pootle_project/views.py (hunk 2 above)
Comment (phlax) [inline pootle/apps/pootle_project/views.py]: ill add a comment

**Pull request merged**